userguide/sharing.rst

```diff
@@ -1092,16 +1092,17 @@ settings:
   in
   :menuselection:`Services --> SMB`.
 
-* Be careful about unchecking the
-  :guilabel:`Browsable to Network Clients` box. When this box is
-  checked (the default), other users will see the names of every share
-  that exists using Windows Explorer, but they will receive a
-  permissions denied error message if they try to access someone
-  else's share. If this box is unchecked, even the owner of the share
-  will not see it or be able to create a drive mapping for the share
-  in Windows Explorer. However, they can still access the share from
-  the command line. Unchecking this option provides limited security
-  and is not a substitute for proper permissions and password control.
+* When the :guilabel:`Browsable to Network Clients` box is checked (the 
+  default), the share will be visible in the server's browse list. If the share 
+  is is flagged as a :guilabel:`home share`, unchecking this box will hide the 
+  [homes] share so that only the dynamically generated home share with the
+  authenticated user's name will be visible (in contrast with [homes]
+  and [<username>] both being visible). 
+
+  The presence of a share in a user's browse list does not grant the user  
```

Comment:
Hard to tell what you mean by "a user's browse list".  Does this mean "Shares that appear in the File Explorer" (or whatever Windows calls it today) "do not necessarily mean the user has read or write permissions to them."?